### PR TITLE
Also added the changes that we discussed today

### PR DESCRIPTION
**Pull request by deaspo**
Added the changes that we worked on Sat, also added the skip. Moved code from from Main to Import Dataset.

### CODE REVIEW

File: instat/dlgImportDataset.Designer.vb [hunk 1]
```diff
@@ -47,12 +47,18 @@ Partial Class dlgImportDataset
         Me.chlStringsAsFactors = New System.Windows.Forms.CheckBox()
         Me.dataFrame = New unvell.ReoGrid.ReoGridControl()
         Me.ucrBase = New instat.ucrButtons()
+        Me.lblFileOpenPath = New System.Windows.Forms.Label()
+        Me.lblLinesToSkip = New System.Windows.Forms.Label()
+        Me.txtFileOPenPath = New System.Windows.Forms.TextBox()
+        Me.nudSkips = New System.Windows.Forms.NumericUpDown()
```
Comment (dannyparsons) [inline instat/dlgImportDataset.Designer.vb]: Have you checked this is the correct naming convention for NumericUpDown?

File: instat/dlgImportDataset.vb [hunk 2]
```diff
@@ -49,19 +54,31 @@ Public Class dlgImportDataset
...
         ucrBase.clsRsyntax.AddParameter("file", Chr(34) & strFilePath & Chr(34))
+        txtFileOPenPath.Text = strFilePath
         txtInputFile.Text = ""
-        For i = 1 To 10 'TODO add skip and get 10 from options
+        'skips
+        If bSkipLines Then
+            For i As Integer = 0 To nudSkips.Value - 1
+                sReader.ReadLine()
+            Next
+        End If
```
Comment (dannyparsons) [inline instat/dlgImportDataset.vb]: Is there any need for this loop as well? I only see a need for the one below that writes in the InputFile text

**Comment by deaspo** [comment]
Removed the first loop that was changing the first preview

File: instat/dlgImportDataset.Designer.vb (hunk 1 above)
Comment (deaspo) [inline instat/dlgImportDataset.Designer.vb]: Yes, this is the currently used naming convention.

**Pull request merged**